commands/push-image.ts

```diff
@@ -35,7 +35,7 @@ export async function pushImage(context?: ImageNode): Promise<void> {
                 command: teleCmdId
             });
 
-            if (imageName.toLowerCase().indexOf('azurecr.io')) {
+            if (imageName.toLowerCase().includes('azurecr.io')) {
```

Comment:
I'd suggest you change the name of the event so we can easily distinguish the old, incorrect ones from the new, correct ones.

No need to run the tool, we'll be able to classify them the first time they appear in Nova.